__tests__/app.test.js

```diff
@@ -141,6 +141,29 @@ describe("API Testing", () => {
 			});
 		});
 	});
+	describe.only("Comments Endpoint", () => {
+		describe("Delete comment by id", () => {
+			test("Delete: 204, should be able to delete a comment by it's id", () => {
+				return request(app).delete("/api/comments/1").expect(204);
+			});
+			test("Delete: 404, responds with an appropriate status and error message when given a non-existent id", () => {
+				return request(app)
+					.delete("/api/comments/1000")
+					.expect(404)
+					.then((response) => {
+						expect(response.body.msg).toBe("comment does not exist");
+					});
+			});
+			test("Delete: 400, responds with an appropriate status and error message when given an invalid id", () => {
+				return request(app)
+					.delete("/api/comments/not-a-comment")
+					.expect(400)
+					.then((response) => {
+						expect(response.body.msg).toBe("Bad request");
+					});
+			});
```

Comment:
Yes great testing here, all 3 tests I expected to see with correct status codes, well done.